Package@swift-5.1.swift

```diff
@@ -125,7 +122,7 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-log", from: "1.0.0"),
         .package(url: "https://github.com/apple/swift-metrics.git", "1.0.0"..<"3.0.0"),
         .package(url: "https://github.com/LiveUI/XMLCoding.git", from: "0.4.1"),
-        .package(url: "https://github.com/amzn/smoke-http.git", from: "2.8.0"),
+        .package(url: "https://github.com/amzn/smoke-http.git", from: "2.7.0"),
```

Comment:
Why do we move from v2.8 to v2.7?

Reply:
2.8.0 of smoke-http adds the experimental async/wait APIs for Swift 5.5 and greater compiler versions.
Swift 5.1 doesn't have access to these APIs so doesn't require the updated version of the dependency. Therefore I reverted changes to the manifest for this compiler version.